bouncer/templates/base.html.jinja2

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+  <html lang=en>
+  <head>
+    <meta charset=utf-8>
+    <title>{% trans %}Hypothesis Direct-Link Bouncer Service{% endtrans %}</title>
```

Comment:
I'd suggest replacing this with a more user-friendly string. Perhaps `Hypothesis: redirecting you...`?